### ISSUE TITLE
[Bug]: Localized fields in blocks

### ISSUE THREAD

**Issue by jremmurd** [pimcore/pimcore#11456]
### Expected behavior

Localized fields in blocks do not throw exceptions, as described in the docs.

See 
* https://pimcore.com/docs/pimcore/current/Development_Documentation/Objects/Object_Classes/Data_Types/Blocks.html  
* Get Values of Localized Block Entries:

### Actual behavior

Getting data from localized fields in blocks throw exceptions 

![image](https://user-images.githubusercontent.com/9052094/154080816-355e2474-7d3f-4488-9769-bb11658e16c7.png)

### Steps to reproduce

Follow the docs 
```

        $object = DataObject\BlockClass::getById(48);
        /** @var  $blockData DataObject\ClassDefinition\Data\Block */
        $blockItems = $object->getBlockElement1();
        /** @var  $firstBlockItem DataObject\Data\BlockElement */
        $firstBlockItem = $blockItems[0];
        $localizedfields = $firstBlockItem["localizedfields"]->getData();
        // after that use it as you would to it with `Pimcore\Model\DataObject\LocalizedField::getLocalizedValue`
```

**Comment by jremmurd** [comment]
@dvesh3 Thx for your research. The issue can be closed, this was a user failure. I put a none existing field into the `->getLocalizedValue($field, $lang)` call - maybe we could improve the error message though.

**Comment by dvesh3** [comment]
I would still change the `getLocalizedValue` method to throw  meaningful exception when a non-exisiting attribute is requested from localizedfields. PR on the way 😉

**Comment by brusch** [comment]
Fixed by #11464
